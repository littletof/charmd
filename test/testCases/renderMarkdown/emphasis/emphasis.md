**This is bold text `**`**

__This is bold text `__`__

*This is italic text `*`*

_This is italic text `_`_

~~This is strikethrough~~ `~~`

**an *italic* in bold**

~~**an *italic* in bold with `striketrough`**~~