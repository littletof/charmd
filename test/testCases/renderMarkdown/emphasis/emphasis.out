[1mThis is bold text [7m ** [27m[22m

[1mThis is bold text [7m __ [27m[22m

[3mThis is italic text [7m * [27m[23m

[3mThis is italic text [7m _ [27m[23m

[9mThis is strikethrough[29m [7m ~~ [27m

[1man [3mitalic[23m in bold[22m

[9m[1man [3mitalic[23m in bold with [7m striketrough [27m[22m[29m
